pkg/build/raw_test.go

```diff
@@ -74,6 +77,7 @@ func TestWriteModifyScript(t *testing.T) {
 	foundContents := string(foundBytes)
 	assert.Contains(t, foundContents, "guestfish --rw -a config-dir/output-image")
 	assert.Contains(t, foundContents, "copy-in "+builder.combustionDir)
+	assert.Contains(t, foundContents, "download /boot/grub2/grub.cfg /tmp/grub.cfg")
```

Comment:
That's covered by the tests in grub_test.go itself. Those are verifying that the template is properly applied. For this test, my only goal is to prove that the grub stuff was called, figuring that if it didn't work, the other test specifically looking for that stuff would fail.